Compilation failed archlinux

**System**
OS: Archlinux
Branch: v0.71.532

**Step to reproduce**
```bash
$ git clone github.com/turanszkij/wickedengine --recursive --branch=v0.71.532
$ cd wickedengine
$ cmake -S . -B build -DCMAKE_BUILD_TYPE=Release
$ cmake --build build -j 12
```

**logs**
```cmake
[  0%] Copying Content
[  5%] Built target FAudio
[ 11%] Built target Utility
[ 20%] Built target LUA
[ 56%] Built target Jolt
[ 56%] Building CXX object WickedEngine/CMakeFiles/WickedEngine_Linux.dir/wiScene_Serializers.cpp.o
[ 56%] Built target Content
/home/tonitch/WickedEngine/WickedEngine/wiScene_Serializers.cpp: In member function ‘void wi::scene::ArmatureComponent::Serialize(wi::Archive&, wi::ecs::EntitySerializer&)’:
/home/tonitch/WickedEngine/WickedEngine/wiScene_Serializers.cpp:879:51: error: ‘boneID’ may be used uninitialized [-Werror=maybe-uninitialized]
  879 |                                 boneCollection[i] = boneID;
/home/tonitch/WickedEngine/WickedEngine/wiScene_Serializers.cpp:877:40: note: ‘boneID’ was declared here
  877 |                                 Entity boneID;
      |                                        ^~~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [WickedEngine/CMakeFiles/WickedEngine_Linux.dir/build.make:930: WickedEngine/CMakeFiles/WickedEngine_Linux.dir/wiScene_Serializers.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:291: WickedEngine/CMakeFiles/WickedEngine_Linux.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

--- a/WickedEngine/wiScene_Serializers.cpp
+++ b/WickedEngine/wiScene_Serializers.cpp
@@ -874,9 +874,7 @@ namespace wi::scene
 			boneCollection.resize(boneCount);
 			for (size_t i = 0; i < boneCount; ++i)
 			{
-				Entity boneID;
-				SerializeEntity(archive, boneID, seri);
-				boneCollection[i] = boneID;
+				SerializeEntity(archive, boneCollection[i], seri);
 			}
 
 			archive >> inverseBindMatrices;

